DiepSam_html/index.html

```diff
@@ -0,0 +1,286 @@
+      <div class="container">
+        <h3>quality policy</h3>
+        <h1>commitment to product quality</h1>
+        <div class="btn-primary">
+          <a href="#">view more</a>
+        </div>
+      </div>
+    </section>
+    <section class="about-us">
+      <div class="container">
+        <div class="section-head">
+          <h3 class="section-title">about us</h3>
+          <div class="logo">
+            <img src="images/icon.png" alt="logo icon">
+          </div>
+        </div>
+        <div class="info-about col-lg-12 col-md-12">
+          <div class="one-block col-lg-4 col-md-4">
+            <h3><span>Interior</span> design consultant</h3>
+            <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit.</p>
+            <img src="images/about1.jpg" alt="information Interior">
+          </div>
+          <div class="one-block col-lg-4 col-md-4">
+            <h3><span>provided</span> interior products</h3>
+            <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit.</p>
+            <img src="images/about2.jpg" alt="information provided">
+          </div>
+          <div class="one-block col-lg-4 col-md-4">
+            <h3><span>policy</span> quality</h3>
+            <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit.</p>
+            <img src="images/about3.jpg" alt="information policy">
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="products">
+      <div class="container">
+        <div class="section-head">
+          <h3 class="section-title">products</h3>
+          <div class="logo">
+            <img src="images/icon.png" alt="logo icon">
+          </div>
+        </div>
+        <div class="tabs col-lg-8 col-md-8">
+          <ul class="tab-link">
+            <li class="tab-link-item"><a href="#">wood</a></li>
+            <li class="tab-link-item"><a href="#">led light</a></li>
+            <li class="tab-link-item"><a href="#">curtain</a></li>
+            <li class="tab-link-item"><a href="#">wood floor</a></li>
+            <li class="tab-link-item"><a href="#">wallpaper</a></li>
+            <li class="tab-link-item"><a href="#">carpet</a></li>
+          </ul>
+        </div>
+      </div>
+      <ul class="list-products-img">
```

Comment:
- Only need name `product-list` and `product-item` 
 - And put it in module `Tabs` ... It inside div `tab-content` .